NOTICE

```diff
@@ -1,4 +1,4 @@
-Datadog datadog-agent
-Copyright 2017 Datadog, Inc.
+Datadog Cloud Foundry Buildpack
+Copyright 2017-Present Datadog, Inc.
```

Comment:
is it 2015 or 2017 the year this project started?

Reply:
earliest commit is 2017